module/executiondatasync/execution_data/execution_data.go

```diff
@@ -16,11 +14,6 @@ type ChunkExecutionData struct {
 	TrieUpdate *ledger.TrieUpdate
 }
 
-type BlockExecutionDataRoot struct {
```

Comment:
Why did we move this?

Reply:
Its now part of the model, since its shared between nodes